source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ACRCustomFeatureFlagResolver.m

```diff
@@ -27,6 +29,10 @@ - (BOOL)boolForFlag:(NSString *)flag
     {
         return YES;
     }
+    if([flag isEqual:isSplitButtonEnabledKey])
```

Comment:
We check for case insensitive string comparison. Also use `isEqualToString` instead.